### DISCUSSION TITLE
1.4

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1159] [General]
I have a question when is tahoma2d 1.4 coming out? And is the symmetry tool the only thing they are adding

**Comment by manongjohn** [comment]
I havent set a date for the 1.4 release yet. I expect it to be before the end of the year. The earliest would be Aug/Sep but dont hold me to that. 😉

You can keep track of what will be included in 1.4 by looking at this list of merged PRs that have a 1.4 label on it: https://github.com/tahoma2d/tahoma2d/pulls?q=is%3Apr+label%3A1.4+is%3Aclosed
 
Anything not in an OT patch release is specific to T2D.

**Comment by Dixxi0509** [comment]
I have a small question is there a way to make a storyboard in tahoma2d

Something like this

https://youtu.be/UjU7BgKYq58

**Comment by manongjohn** [comment]
True Storyboarding, no. Not yet anyway. Maybe in the future. The Preproduction Board from OT will be included in 1.4.  It's in the nightly version already.

In the list of 1.4 changes I provided earlier, the OT Patches entries contain the list of features/fixes ported over from OT into T2D. Most things from OT are ported over. Read each one to see what is included.

**Comment by Dixxi0509** [comment]
Oh I understand I'll wait :3
